delft3dworker/management/commands/containersync_sceneupdate.py

```diff
@@ -15,4 +19,48 @@ class Command(BaseCommand):
     help = "sync containers with container and scene model"
 
     def handle(self, *args, **options):
-        self.stdout.write("containersync_sceneupdate skeleton")
+
+        # Loop over non empty celery_task_ids in containers
+        # celery_set = set(
+            # Container.objects.exclude(celery_id__exact='').values_list('celery_id', flat=True))
+
+        # retrieve containers from docker
+        containers_docker = get_docker_ps()
```

Comment:
Should this not be called like so:
```
try:
    containers_docker = get_docker_ps.delay(timeout=30)
except celery.exceptions.TimeoutError as e:
    logging.exception("get_docker_ps timed out (30 seconds)")

if containers_docker not is None:
    ....
```

It seems like calling it directly will not get the remote `docker ps`, but an unintended local one.